frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -448,24 +452,27 @@ export const createOrUpdateResources = async (
       generatedImageStreamName ? 'create' : verb,
       generatedImageStreamName,
     ),
-    createOrUpdateBuildConfig(
-      formData,
-      imageStream,
-      dryRun,
-      _.get(appResources, 'buildConfig.data'),
-      verb,
-      generatedImageStreamName,
-    ),
   );
+  if (!pipeline.enabled) {
+    requests.push(
+      createOrUpdateBuildConfig(
+        formData,
+        imageStream,
+        dryRun,
+        appResources?.buildConfig?.data,
+        verb,
+        generatedImageStreamName,
+      ),
+    );
+  } else if (pipeline.template && !dryRun) {
+    const newPipeline = await createPipelineForImportFlow(formData);
+    requests.push(createPipelineRunForImportFlow(formData, newPipeline));
```

Comment:
Also, if this fails (which I don't think is possible today) we'll need to not cancel the submit flow. I've created another story (https://issues.redhat.com/browse/ODC-5048) to chase this down as it may require hacking up the resource in order to test it. No sense in holding up this PR for potentially a non-starter flow.